client/payments/payment-recommendations.scss

```diff
@@ -59,6 +59,10 @@
 		margin-left: $gap-small;
 	}
 
+	.woocommerce-list__item-before img {
+		max-width: 96px;
+	}
+
```

Comment:
Could we also center the images vertically, there seems to be quite a bit of space below some of them:
<img width="1253" alt="Screen Shot 2022-02-07 at 1 18 43 PM" src="https://user-images.githubusercontent.com/2240960/152849954-03494e3f-2456-4db4-8193-c813ca2c9d8e.png">

Reply:
Seems like a reasonable change.  Updated to the whole row aligning center (including the action button) to match the suggestions in the task list.